sys/dev/bce/if_bce.c

```diff
@@ -529,6 +529,7 @@ MODULE_DEPEND(bce, miibus, 1, 1, 1);
 
 DRIVER_MODULE(bce, pci, bce_driver, bce_devclass, NULL, NULL);
 DRIVER_MODULE(miibus, bce, miibus_driver, miibus_devclass, NULL, NULL);
+MODULE_PNP_INFO("U16:vendor;U16:device;D:#", pci, bce, bce_devs, sizeof(bce_devs[0]), nitems(bce_devs) - 1);
```

Comment:
The descriptor string is wrong.  `bce_type` is:

```
struct bce_type {
        u_int16_t bce_vid;
        u_int16_t bce_did;
        u_int16_t bce_svid;
        u_int16_t bce_sdid;
        const char *bce_name;
};
```

So you've got vendor and device correct, but the next part should be "V16:subvendor;V16:subdevice;D:#".

Note that the subvendor/subdevice matching is `V` rather than `U` — PCI_ANY_ID (0xffff) is used for some entries and means, match any value in this field.  That's what the 'V' character signifies as well — `0xffff` matches any.

The bus, module, table entry size and count looks correct.